src/frequenz/sdk/actor/power_distributing/_battery_status.py

```diff
@@ -12,10 +12,13 @@
 from enum import Enum
 from typing import Iterable, Optional, Set, TypeVar, Union
 
+# pylint: disable=no-name-in-module
```

Comment:
yes,  I'm guessing because some protobuf dep version changed in the api repo.  That's why I put them all in a single comment,  so it can be removed easily once we figure out the issue 🤞🏽

Reply:
I actually have a theory now:

Pylint doesn't look at the type annotations,  it just looks at the code other other modules to see if the desired functions are there.  This works only if the names are defined in python code in the modules.  But protoc has not been doing that for a while now,  and is just using dynamic module creation during import,  and we appear to have to just use the type hints to know what's going on.  If you look at one of the generated files,  this becomes clear,  like `.nox/pylint/lib/python3.11/site-packages/frequenz/api/microgrid/inverter_pb2.py`

And it was working until now, because we were using a very old version of protoc in the api repo.